app/templates/friendview.html

```diff
@@ -0,0 +1,31 @@
+{% extends "base.html" %}
+{% block head %}
+{{ super() }}
+<link href="{{ url_for('static', path='/friendview.css') }}" rel="stylesheet">
+{% endblock %}
+{% block content %}
+<div class="container mt-4">
+    <h1>Friends View </h1>
+</div>
+<form method="GET" action="/friendview/">
+    <div class="mb-3 friendview"><br><br>
+        <label for="my_friend">My friend</label><br><br>
```

Comment:
You should use margin to space between things.

Reply:
I have changed those and cancel the <br>